odoo/addons/base/populate/res_user.py

```diff
@@ -30,13 +34,24 @@ def generate_partner_id(iterator, *args):
         def get_company_ids(values, **kwargs):
             return [(6, 0, [values['company_id']])]
 
+        def get_groups_id(values, counter, random):
+            groups = []
+            group = random.choices(['group_user', 'group_portal'], [0.8, 0.2])
+            if group == 'group_user':
```

Comment:
this condition will never be true because `random.choices()` always returns a list.
should be `if group[0] == 'group_user`